src/server.c

```diff
@@ -1107,7 +1107,7 @@ void removeClientFromMemUsageBucket(client *c, int allow_eviction) {
  *
  * Note: This function filters clients of type no-evict, primary or replica regardless
  * of whether the eviction is enabled or not, so the memory usage we get from these
- * types of clients via the INFO command may be out of date.
+ * types of clients via the INFO command may be out-of-date.
```

Comment:
`out of date` is correct here, `out-of-date` as a compound adjective is only valid before a noun